### PR TITLE
Export strings for Denim

### PR DESCRIPTION
**Pull request by bkmunar**

### CODE REVIEW

**Comment by bkmunar** [comment]
@flodolo @Delphine

File: en-US/focus-ios.xliff [hunk 1]
```diff
@@ -71,6 +71,11 @@
         <target>Help</target>
         <note>Label for row in About screen</note>
       </trans-unit>
+      <trans-unit id="About.rowPrivacy">
+        <source>Privacy Notice</source>
+        <target>Privacy Notice</target>
+        <note>Label for row in About screen</note>
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: Nit for future strings: this sounds like a developer comment (I guess rows is what you use to create the layout?)

My feeling is this is a "Link to Privacy Notice in the About screen", or something along that line

File: en-US/focus-ios.xliff [hunk 2]
```diff
@@ -201,6 +206,11 @@
         <target>Browse like no one’s watching.</target>
         <note>Message label on the first run screen</note>
       </trans-unit>
+      <trans-unit id="Focus.handoffSyncing">
+        <source>Apple Handoff is syncing</source>
+        <target>Apple Handoff is syncing</target>
+        <note>Title for the loading screen when the handoff of clipboard delays Focus launch.</note>
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: Looks like Handoff is a name that should not be localized
https://support.apple.com/it-it/HT204681
https://support.apple.com/es-es/HT204681

We should add that into the localization note: "Title for the loading screen when the handoff of clipboard delays Focus launch. Handoff should not be localized, see https://support.apple.com/HT204681".

File: en-US/focus-ios.xliff [hunk 3]
```diff
@@ -266,6 +276,16 @@
         <target>I Understand</target>
         <note>Button label for accepting Content blocker alert</note>
       </trans-unit>
+      <trans-unit id="Settings.detailTextSendUsageData">
+        <source>Mozilla strives to collect only what we need to provide and improve Focus for everyone. </source>
+        <target>Mozilla strives to collect only what we need to provide and improve Focus for everyone. </target>
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: There's an extra space at the end of the string

File: en-US/focus-ios.xliff [hunk 4]
```diff
@@ -339,12 +359,15 @@
       <trans-unit id="Settings.toggleSendAnonymousUsageData">
         <source>Send anonymous usage data</source>
         <target>Send anonymous usage data</target>
-        <note>Label for Send Anonymous Usage Data toggle on main screen</note>
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: Why did we lose this note? I don't see any change in the code that should cause this :-\

File: en-US/focus-ios.xliff [hunk 5]
```diff
@@ -339,12 +359,15 @@
...
       </trans-unit>
       <trans-unit id="Settings.toggleSendAnonymousUsageDataSubtitle">
         <source>Learn more</source>
         <target>Learn more</target>
-        <note>Subtitle for Send Anonymous Usage Data toggle on main screen</note>
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: Same here

File: en-US/focus-ios.xliff [hunk 6]
```diff
@@ -396,6 +419,11 @@
         <target>Open in %@</target>
         <note>Title for action sheet item to open the current page in another application. Placeholder is the name of the application to open the current page.</note>
       </trans-unit>
+      <trans-unit id="contextMenu.clipboardLink">
+        <source>Link you copied:</source>
+        <target>Link you copied:</target>
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: Do you have a screenshot or UX mockup of how this string will be used? Is it really a context menu? How do you plan to insert the link?

If it's only one line, there should be a placeholder %@ for the link.

**Comment by flodolo** [comment]
@bkmunar (or @boek)
Can you please prioritize this in your morning? I'm 9 hours away, your 9am it's my 6pm on a Friday, otherwise this will slip once again out of schedule

File: en-US/focus-ios.xliff (hunk 4 above)
Comment (flodolo) [reply en-US/focus-ios.xliff]: See https://github.com/mozilla-mobile/focus-ios/issues/395

**Comment by flodolo** [comment]
I'm closing this one, since there is a PR with fixes waiting to land in 
https://github.com/mozilla-mobile/focus-ios/pull/398

**Pull request closed without merging**